src/interfaces.ts

```diff
@@ -28,6 +28,10 @@ export interface SelectAll extends SelectOne {
   limit?: number
 }
 
+export interface Subquery extends SelectAll {
+  alias?: string
```

Comment:
maybe this alias field could be moved into the `Join` interface, that way people using the "simple" join could also benefit from the alias